guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,230 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
+
+{Project} and {SmartProxy} are supported on both {EL}8 and {EL}9.
+You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9. 
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to upgrade as well as to help detect and resolve issues that could prevent you from upgrading successfully.
+
+.Prerequisites
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
```

Comment:
I would add a similar statement for upstream. There's a section in upstream Release Notes called "Upgrade warnings".